.storybook/colors.stories.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component, Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import styled from 'styled-components';
+
+import Heading from '../src/components/Heading';
+import { contrastingColor } from '../src/utils';
+import { COLORS as guppyColors } from '../src/constants';
+import { default as materialColors } from 'material-colors/dist/colors';
```

Comment:
Ahh, I might have been unclear earlier D:

What I meant was that we should quickly grep through the project to find the 10ish colors we're likely already using, as a subset of all the colors in `COLORS`.

I have mixed feelings about what you've done. On the one hand it seems convenient to have additional colors available, although I wonder if we want to encourage people to use additional colors (I think we likely have more than enough already, even for edge-case things);

Sorry for not being clearer :( FWIW I really like how you handled the `info` distinction and the ColorSwatch components and such. I think we can probably use exactly the same structure, just with different input colors?

I can also be the one who scans through and finds the "common" ones if you prefer! I don't wanna trouble you any more with this PR haha.

Reply:
No problem 😁 I did think it was a little strange to include the entire Material palette, I definitely think keeping things focused makes more sense. Now I see what you were saying, there are definitely quite a few colors included that only appear once or not at all.

I'll update this tomorrow according to those specs and then you can review the common colors and make sure I got everything you had in mind!

Reply:
Yay, thanks for understanding! Sounds good.

(I just noticed I ended one of my paragraphs in my comment in a semicolon, hah.)